packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -265,6 +265,19 @@ export type CallWithChatCompositeOptions = {
       url: string;
     };
   };
+  /* @conditional-compile-remove(spotlight) */
+  /**
+   * Options for settings related to spotlight.
+   */
+  spotlight?: {
+    /**
+     * Flag to hide the menu buttons to start and stop spotlight for remote participants and menu button
+     * to start spotlight for the local participant. The menu button to stop spotlight for the local
+     * participant will not be hidden.
+     * @defaultValue false
+     */
+    hideSpotlight?: boolean;
```

Comment:
Based on the code comment, `hideSpotlight` might need to be renamed. Would `onlyExitSpotlight` be better?

Reply:
Unsure on this, onlyExitSpotlight drives better meaning, but is also perhaps confusing at first usage? 😅
`hideSpotlight` with a `@remark` that is cannot hide the stop spotlight from a local user could also work?
Surprised we have to show the stopSpotlight for the local user, if Contoso builds their own app they won't have to show that.
Other alternative could be 
```
spotlightSettings: {
  hideRemoteSpotlight: boolean;
  hideLocalStartSpotlight: boolean
}
```
? But I dont love that either - approving PR as I think any of these work!